packages/react-composites/tests/app/lib/MockCallAdapter.ts

```diff
@@ -178,11 +178,11 @@ function populateViewTargets(state: MockCallAdapterState): CallAdapterState {
  * @param str - input string
  * @returns
  */
-const createMockHTMLElement = (str?: string): HTMLElement => {
+const createMockHTMLElement = (str?: string, widthPx?: number, heightPx?: number): HTMLElement => {
   const mockVideoElement = document.createElement('div');
   mockVideoElement.innerHTML = '<span />';
-  mockVideoElement.style.width = decodeURIComponent('100%25');
-  mockVideoElement.style.height = decodeURIComponent('100%25');
+  mockVideoElement.style.width = widthPx ? `${widthPx}` : decodeURIComponent('100%25');
```

Comment:
the default is `height: 100%; width: 100%` to fill the container and completely cover the video, why is the manual width and height needed here?

Reply:
We don't need this code right now, removed it.